ledger-bridge.js

```diff
@@ -138,17 +201,12 @@ export default class LedgerBridge {
           return err.metaData.type
         }
 
-        if (isStringError(err)) {
-          // Wrong app logged into
-          if (err.includes('6804')) {
+        if (isWrongAppError(err) || (isStringError(err) && err.includes('6804'))) {
```

Comment:
Nit: it might make sense to move the second half of this into the `isWrongAppError` as well. I had to cross-reference these to ensure the correct error code was being used in both places.

Reply:
Good point!